### DISCUSSION TITLE
no output for landsat level2 for some tiles

### DISCUSSION THREAD

**Discussion by belford33** [davidfrantz/force#319] [Q&A]
Hallo David, 
I was updating my existing Landsat data archive (L8, L9) for two tiles for the years 2022-2023. One tile gives me correct data (192/027) while there is no updated data for the other tile (except when i force it to 100% cc, then i get an image with nodata values). The log files are all the same with data coverage: 0 and cloud coverage 100%. I first thought it might be the DEM coverage but it is definitely large enough (and it all ran smoothly before). I attach here a log files and the parameter files, I am using FORCE DEV. 

Any idea what might be the cause?  
Thanks a lot for your support,
best,
Thomas

[192027.txt](https://github.com/davidfrantz/force/files/14945419/192027.txt)
[193027.txt](https://github.com/davidfrantz/force/files/14945421/193027.txt)
[LC08_L1TP_193027_20230907_20230912_02_T1.txt.log](https://github.com/davidfrantz/force/files/14945422/LC08_L1TP_193027_20230907_20230912_02_T1.txt.log)

Update: I did another run without the DEM and I do get also results for the 193027 tile. Yet, I don't understand why this is happening and I would like to use the topographic correction. The DEM is correct and large enough. 
The only notable difference between the two tiles is the difference in UTM zone (UTM 33 works, UTM 32 doesn't). The DEM is in geographic coordinates.

**Comment by davidfrantz** [comment]
Dear Thomas,

I don't see any issue on my end and could not reproduce the behaviour:


```
FORCE Level 2 Processing System
-----------------------------------------------------------

Start of processing: 2024-04-11 14:01:34
Image: /home/ahsoka/frantz/temp/thomas/LC08_L1TP_193027_20230907_20230912_02_T1.tar
Unpacking tar container successful

Start core processing
-----------------------------------------------------------

dc:  67.22%. wc:   1.02%. sc:   0.57%. cc:   7.46%. AOD: 0.0783. # of targets: 72/5429.  1 product(s) written. Success! Processing time: 02 mins 39 secs

-----------------------------------------------------------
Core processing signaled DONE

End of processing: 2024-04-11 14:04:14
May the FORCE be with you!
```

Are you running the job through Docker, or is this a custom installation?

You could try using the debug container (dev-debug) and inspect the individual output layers.

Best,
David
